rero_ils/modules/vendors/mappings/v6/vendors/vendor-v0.0.1.json

```diff
@@ -47,6 +47,34 @@
         },
         "default_contact": {
           "properties": {
+            "contact_person": {
+              "type": "text"
+            },
+            "city": {
+              "type": "keyword"
+            },
+            "country": {
+              "type": "keyword"
+            },
+            "email": {
+              "type": "keyword"
+            },
+            "phone": {
+              "type": "keyword"
+            },
+            "postal_code": {
+              "type": "keyword"
+            },
+            "street": {
+              "type": "text"
+            }
+          }
+        },
+        "order_contact": {
+          "properties": {
+            "contact_person": {
+              "type": "text"
```

Comment:
`text` ? Will you search on "Give me all vendors where contact person contains 'John' ?" In my sense `keyword` should be better